web/src/components/molecules/View/viewsMenu.tsx

```diff
@@ -14,6 +18,7 @@ interface Props {
   currentView: CurrentView;
   onViewCreateModalOpen: () => void;
   onViewSelect: (key: string) => void;
+  onUpdateViewsOrder: (viewIds: string[]) => void;
```

Comment:
It might be better to write it this way.
```suggestion
  onUpdateViewsOrder: (viewIds: string[]) => Promise<void>;
```